packages/cli/src/commands/pw-test.ts

```diff
@@ -0,0 +1,380 @@
+
+
+    const location = await prepareRunLocation(checklyConfig.cli, {
+      runLocation: runLocation as keyof Region,
+      privateRunLocation,
+    }, api, config.getAccountId())
+    const reporterTypes = prepareReportersTypes(reporterFlag as ReporterType, checklyConfig.cli?.reporters)
+    const { data: account } = await api.accounts.get(config.getAccountId())
+    const { data: availableRuntimes } = await api.runtimes.getAll()
+    const testEnvVars = await getEnvs(envFile, env)
+
+
+    const project = await parseProject({
+      directory: configDirectory,
+      projectLogicalId: checklyConfig.logicalId,
+      projectName: testSessionName ?? checklyConfig.projectName,
+      repoUrl: checklyConfig.repoUrl,
+      includeTestOnlyChecks: true,
+      checkMatch: checklyConfig.checks?.checkMatch,
+      ignoreDirectoriesMatch: checklyConfig.checks?.ignoreDirectoriesMatch,
+      checkDefaults: checklyConfig.checks,
+      availableRuntimes: availableRuntimes.reduce((acc, runtime) => {
+        acc[runtime.name] = runtime
+        return acc
+      }, <Record<string, Runtime>> {}),
+      defaultRuntimeId: account.runtimeId,
+      verifyRuntimeDependencies: false,
+      checklyConfigConstructs,
+      playwrightConfigPath,
+      include: checklyConfig.checks?.include,
+      playwrightChecks: [playwrightCheck],
+      checkFilter: check => {
+        if (Object.keys(testEnvVars).length) {
+          check.environmentVariables = check.environmentVariables
+            ?.filter((envVar: any) => !testEnvVars[envVar.key]) || []
+          for (const [key, value] of Object.entries(testEnvVars)) {
+            check.environmentVariables.push({
+              key,
+              value,
+              locked: true,
+            })
+          }
+        }
+        return true
+      }
+    })
+
+    this.style.actionSuccess()
+
+    this.style.actionStart('Validating project resources')
+
+    const diagnostics = new Diagnostics()
+    await project.validate(diagnostics)
+
+    for (const diag of diagnostics.observations) {
+      if (diag.isFatal()) {
+        this.style.longError(diag.title, diag.message)
+      } else if (!diag.isBenign()) {
+        this.style.longWarning(diag.title, diag.message)
+      } else {
+        this.style.longInfo(diag.title, diag.message)
+      }
+    }
+
+    if (diagnostics.isFatal()) {
+      this.style.actionFailure()
+      this.style.shortError(`Unable to continue due to unresolved validation errors.`)
+      this.exit(1)
+    }
+
+    this.style.actionSuccess()
+
+    this.style.actionStart('Bundling project resources')
+    const projectBundle = await (async () => {
+      try {
+        const bundle = await project.bundle()
+        this.style.actionSuccess()
+        return bundle
+      } catch (err) {
+        this.style.actionFailure()
+        throw err
+      }
+    })()
+
+    const checkBundles = Object.values(projectBundle.data.check)
+
+    if (this.fancy) {
+      ux.action.stop()
+    }
+
```

Comment:
Can you remove this? I know the `test` command also has it (by mistake it looks like) and you've just copied it, but there is no corresponding start for this stop action, so it should be removed. Also, it will potentially mean removing the `ux` import.

```suggestion
```